generate/genfile/genfile.go

```diff
@@ -127,6 +141,23 @@ func Load(rootdir string, sm stack.Metadata, globals stack.Globals) (StackFiles,
 			return StackFiles{}, errors.E(ErrContentEval, err)
 		}
 
+		condition := true
```

Comment:
we should not evaluate the content block before evaluating the `condition`.
If the condition is false, then the content must not be evaluated.
The condition can check for the existence of a variable that will be used in the content block, so with current approach some valid use cases would not be possible.